digitalocean/resource_digitalocean_kubernetes_node_pool.go

```diff
@@ -66,9 +66,23 @@ func nodePoolSchema() map[string]*schema.Schema {
 		},
 
 		"node_count": {
-			Type:         schema.TypeInt,
-			Required:     true,
-			ValidateFunc: validation.IntAtLeast(1),
+			Type:     schema.TypeInt,
+			Required: true,
+		},
```

Comment:
Is it no longer required to have a minimum of 1 instance in the node pool?

Reply:
Node pools can be empty but the cluster needs to be non-empty. Going to remove this and PR it separately.